controller/system_rollout_controller.go

```diff
@@ -1755,18 +1776,26 @@ func tagLonghornLastSystemRestoreAnnotation(systemRestoredURL, systemRestoredAt
 		annos = map[string]string{}
 	}
 
+	hasAnno, err := c.hasSystemRestoredAnnotation(obj, false)
```

Comment:
Why having this case? the resources have been restored before the system backup? Should we just override?

Reply:
case:
1. Resource CR-1 gets created (tagged) and CR-2 failed (un-tagged).
1. Controller tries to restore resources again on the ticker.

Before change:
1. Resource CR-1 gets tagged again with skipped annotation because it's not created and the backup CR-1 and existing CR-1 are identical.
1. Resource CR-1 has the wrong annotation.

After change:
1. Resource CR-1 will not get tagged again.
1. Resource CR-1 has the correct annotation.